### DISCUSSION TITLE
[Feature] Display Icon in thumbnail if RAW Photo

### DISCUSSION THREAD

**Discussion by sirebellum** [immich-app/immich#6267] [Feature Request]
### The feature

Hi there,

I recently started using Immich to store my (huge) trove of images. It's been great with all the AI features and customization. I'd like to see a feature I used to use in my google photos app where the thumbnail would display whether or not a file was a raw (ARW, RAW, etc.) file. This helped whenever I've edited a file and have multiple versions, but the jpg is the edited version.

I believe an icon already exists for this (Icons.raw_on_rounded).

Please let me know your thoughts, I'd also be happy to do some dev work on this.

Regards

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by sirebellum** [comment]
@shenlong-tanwen What are the odds I could get some pointers here 👀

**Comment by alextran1502** [comment]
There isn't a good UI to display the icon without clustering the thumbnail. I want to keep the timeline thumbnail simple to browse photos, not to find photos.

An alternative approach is for the search enhancement; we have a mechanism to filter out raw photos format

**Reply by sirebellum** [reply]
Fair enough, searching for filetype seems reasonable.

**Comment by sirebellum** [comment]
Close in favor of search enhancement for finding files by filetype
